Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -169,6 +186,17 @@ cpu_timer0_isr(void)
     PieCtrlRegs.PIEACK.all = PIEACK_GROUP1;
 }
 
+__interrupt void
+cpu_timer1_isr(void)
```

Comment:
If this interruption is used for ADC maybe it should be added to AnalogInputMgr, in case of the timer0, it has been moved to th scheduler file.

Reply:
I have added it to the analog manager